plonetheme/onegovbear/theme/scss/form.scss

```diff
@@ -25,6 +25,7 @@ $input-error-border-color: transparentize($danger-color, .5) !default;
   padding: 0.5em;
   width: 20em;
   outline: none;
+  box-sizing: border-box;
```

Comment:
This as an effect on all forms on a site, not only the ones in a overlay form. 

imho this could be kind of destructive to other forms, like contact, feedback, search, controlpanel, etc. 

cc @jone This should be applied only to form fields in an overlay!